content/tracing/_index.md

```diff
@@ -2,75 +2,113 @@
 title: APM (Tracing)
 kind: Documentation
 description: Instrument your code to improve performance
+further_reading:
+- link: "/tracing/languages"
+  tag: "Documentation"
+  text: Instrument your code
+- link: "/tracing/services"
+  tag: "Documentation"
+  text: Analyse your services
 ---
 
 ## Overview
 
 Datadog's integrated APM tool eliminates the traditional separation between infrastructure and application performance monitoring. This not only provides greater visibility, but also allows you to see the relationship between application code and the underlying infrastructure.
 
-## Getting started
-
-Datadog APM is offered as an upgrade to our Pro and Enterprise plans. A free 14-day trial is available.  Registered users can visit the [APM page of the Datadog application](https://app.datadoghq.com/apm/home) to get started.
+Datadog APM is offered as an upgrade to our Pro and Enterprise plans. A free 14-day trial is available.
+Registered users can visit the [APM page of the Datadog application](https://app.datadoghq.com/apm/home).
 
+## Setup
```

Comment:
I think the "Installation" and "Instrument your application" sections can be collapsed.  This is a little repetitive.

Reply:
👍